tests/smoke-tests/test-proxy-consistency.ts

```diff
@@ -15,14 +15,15 @@ const wssUrl = process.env.WSS_URL || null;
 const relayWssUrl = process.env.RELAY_WSS_URL || null;
 
 // TEMPLATE: Give suitable name
-describeSmokeSuite(`Verify number of proxies per account`, { wssUrl, relayWssUrl }, (context) => {
+describeSmokeSuite(`Verify account proxies created`, { wssUrl, relayWssUrl }, (context) => {
   // TEMPLATE: Declare variables representing the state to inspect
   //           To know the type of the variable, type the query and the highlight
   //           it to see
   //           Ex: context.polkadotApi.query.proxy.proxies.entries()
   //             Displays PalletProxyProxyDefinition
   //           Then add the type in the import from "@polkadot/types/lookup"
   const proxiesPerAccount: { [account: string]: PalletProxyProxyDefinition[] } = {};
+  let proxyAccList = [];
```

Comment:
FYI @timbrinded:
We usually try to do some functional programming (not too strict on it) and we also try to use `const` variable whenever we can (here it is an array and you use push so the `let` could be `const`). It makes the code more readable (knowning if a variable will change or not later in the code)

Reply:
This was meant to be a constant, I always try to use them where possible. I think I must have left it as a let by accident.

Do you have any suggestions of what I've done here that can be improved with a functional flavour?

Reply:
I think I know what you mean, it's a bit gross to have all that state collected in the beforeAll() as a global. For this pr I wanted to use what was already written, but can rewrite this whole file to be purer if thats a priority over more new test cases.